docs/root/configuration/listeners/network_filters/_include/wasm_filters.yaml

```diff
@@ -0,0 +1,24 @@
+static_resources:
+  listeners:
+    - address:
+        socket_address:
+          address: 0.0.0.0
+          port_value: 8000
+      filter_chains:
+        - filters:
+          - name: envoy.filters.network.wasm
+            typed_config:
+              "@type": type.googleapis.com/envoy.extensions.filters.network.wasm.v3.Wasm
+              config:
+                name: "my_plugin"
+                vm_config:
+                  runtime: "envoy.wasm.runtime.v8"
+                  code:
+                    local:
+                      filename: "/etc/envoy_filter_http_wasm_example.wasm"
```

Comment:
im thinking this might be the issue - iirc we have a wasm bin in the examples on a different path, checking...

Reply:
yep, checking the other example wasm yaml files i think `lib/envoy_filter_http_wasm_example.wasm` should work

Reply:
Hmm, I tried `filename: "lib/envoy_filter_http_wasm_example.wasm"` and `filename: "../../../../../../../examples/wasm-cc/lib"` but both does not work...

Reply:
it needs to be just `lib/envoy_filter_http_wasm_example.wasm`

Reply:
also you will need a complete configuration - ie including a cluster

checking for existing docs that use a wasm example, there is one here `docs/root/configuration/http/http_filters/wasm_filter.rst`

if it has the required code, you could use that one (in that case you do need to figure out the `../../`)

Reply:
im wondering if this is related #18127 given the error message relates to `customStatNamespaces`